jaeger-spark-dependencies-elasticsearch/src/main/java/io/jaegertracing/spark/dependencies/elastic/ElasticsearchDependenciesJob.java

```diff
@@ -157,7 +157,7 @@ private static String getSystemPropertyAsFileResource(String key) {
 
     String indexPrefix = builder.indexPrefix;
     if (indexPrefix != null) {
-      indexPrefix = String.format("%s:", indexPrefix);
+      indexPrefix = String.format("%s-", indexPrefix);
```

Comment:
This is not all right. I will change it to read from two indices - one separator `-` other `:`. And write only to the index with separator `-`